### PR TITLE
Fix libvirt coreos cluster

### PR DESCRIPTION
**Pull request by jmastr**

### CODE REVIEW

**Comment by jmastr** [comment]
I signed it!

**Comment by jmastr** [comment]
Signing attempt number 2!

**Comment by brendandburns** [comment]
@jmastr I don't see your github user in our system.  Try again?

thanks!
--brendan

**Comment by jmastr** [comment]
I tried it and received an email, that an executive or a lawyer from our company had to sign the CLA. That was done yesterday.

**Comment by jmastr** [comment]
@brendandburns ping!

**Comment by mirkoboehm** [comment]
And we signed the CLA a third time, this time me. I think this should be resolved by now.

**Comment by brendandburns** [comment]
CLA validated.  To make this automatic, you need to set up a google group for the company and make sure everyone covered by the CLA is added to it.

Thanks!
--brendan

**Comment by brendandburns** [comment]
LGTM

You need to run
`./hack/update-generated-docs.sh`

Thanks!
--brendan

@k8s-bot ok to test

File: cluster/libvirt-coreos/util.sh [hunk 1]
```diff
@@ -130,12 +134,11 @@ function initialize-pool {
...
       virsh vol-delete coreos_base.img --pool $POOL 2> /dev/null || true
-      mv "$ROOT/coreos_production_qemu_image.img" "$POOL_PATH/coreos_base.img"
   fi
-  # if ! virsh vol-list $POOL | grep -q coreos_base.img; then
-  #     virsh vol-create-as $POOL coreos_base.img 10G --format qcow2
-  #     virsh vol-upload coreos_base.img "$ROOT/coreos_production_qemu_image.img" --pool $POOL
-  # fi
+  if ! virsh vol-list $POOL | grep -q coreos_base.img; then
+      virsh vol-create-as $POOL coreos_base.img 10G --format qcow2
+      virsh vol-upload coreos_base.img "$ROOT/coreos_production_qemu_image.img" --pool $POOL
```
Comment (lhuard1A) [inline cluster/libvirt-coreos/util.sh]: As far as I remember, `virsh vol-upload` was *much* slower than the `mv`.
If it is still the case, why not keeping the faster `mv` ?

Comment (thomasfricke) [inline cluster/libvirt-coreos/util.sh]: The `mv` does not work everywhere.

The old version mixed file transfers and libvirt storage/volume commands, which might work on some machines, but not everywhere.

The image update is done only once iff a new version is uploaded.

If this is crucial to someone, pure file transfer method could be provided, but I would stay away from this approach.

Does this help?

**Comment by jmastr** [comment]
@brendandburns Hi, I made the commit you requested. Regarding the CLA: @thomasfricke is part of our CLA google group.

Regards

**Comment by brendandburns** [comment]
@k8s-bot test this please

**Pull request merged**